### PR TITLE
Update list of indexed collections

### PR DESCRIPTION
**Pull request by dmsimard**
- Add collections that are in the Ansible package but were missing
- Update collection repositories for those that have moved

I tried keeping the existing extra keys (Regex, NewCollection and Merge) where they existed but they are ommitted from new additions.

Some notes as I worked through this:

Indexed but not currently in the Ansible package:
- ansible-collections/community.asa
- ansible-collections/community.molecule
- ansible-collections/community.cassandra
- ansible-collections/community.qradar (though ibm.qradar is in the package)
- ansible-collections/community.yang
- ansible-collections/vmware.vmware_rest
- ansible-collections/checkpoint (though check_point.mgmt via CheckPointSW/CheckPointAnsibleMgmtCollection)
- ansible-collections/ibm.spectrum_virtualize
- ansible-collections/pureport

Repositories that have moved:
- ansible-collections/vmware moved to ansible-collections/community.vmware
- ansible-collections/ansible_collections_google moved to ansible-collections/google.cloud
- ansible-collections/openvswitch moved to ansible-collections/openvswitch.openvswitch
- ansible-collections/ios moved to ansible-colletions/cisco.ios
- ansible-collections/ios moved to ansible-colletions/cisco.iosxr
- ansible-collections/grafana moved to ansible-collections/community.grafana
- ansible-collections/dellemc_networking.os10 moved to ansible-collections/dellemc.os10
- ansible-collections/dellemc_networking.os9 moved to ansible-collections/dellemc.os9
- ansible-collections/dellemc_networking.os6 moved to ansible-collections/dellemc.os6
- ansible-collections/eos moved to ansible-collections/arista.eos
- ansible-collections/frr moved to ansible-collections/frr.frr
- ansible-collections/netcommon moved to ansible-collections/ansible.netcommon
- ansible-collections/vyos moved to ansible-collections/vyos.vyos
- ansible-collections/junos moved to ansible-collections/junipernetworks.junos

The diff is a bit unwieldy and thus I propose a diff of only the Repo key (sorted):
```
> diff --side-by-side --suppress-common-lines /tmp/new /tmp/old
    Repo: ansible-aci					      |	    Repo: ansible_collections_google
    Repo: ansible-collection-cloudscale			      <
    Repo: ansible-collection-cloudstack			      <
    Repo: ansible-collection-exoscale			      <
    Repo: ansible-collection-icinga-director		      <
    Repo: ansible-collection-vultr			      <
    Repo: ansible-conjur-collection			      <
    Repo: ansible-galaxy-fortimanager-collection	      <
    Repo: ansible-galaxy-fortios-collection		      <
    Repo: ansible-infinidat-collection			      <
    Repo: ansible-meraki				      <
    Repo: ansible_modules				      <
    Repo: ansible-mso					      <
    Repo: ansible.netcommon				      <
    Repo: ansible-nso					      <
    Repo: ansible.posix					      <
    Repo: ansible-security-automation-collection	      <
    Repo: ansible-ucs					      <
    Repo: arista.eos					      <
    Repo: awx						      <
    Repo: CheckPointAnsibleMgmtCollection		      |	    Repo: checkpoint
    Repo: chocolatey-ansible				      <
    Repo: cisco.ios					      <
    Repo: cisco.iosxr					      <
    Repo: community.fortios				      <
    Repo: community.google				      |	    Repo: community.grafana
    Repo: community.hashi_vault				      <
    Repo: community.hrobot				      <
    Repo: community.vmware				      <
    Repo: dellemc-openmanage-ansible-modules		      |	    Repo: dellemc_networking.os10
    Repo: dellemc.os10					      |	    Repo: dellemc_networking.os6
    Repo: dellemc.os6					      |	    Repo: dellemc_networking.os9
    Repo: dellemc.os9					      |	    Repo: eos
    Repo: f5-ansible					      <
    Repo: FlashArray-Collection				      <
    Repo: FlashBlade-Collection				      <
    Repo: frr.frr					      |	    Repo: frr
    Repo: gluster-ansible-collection			      <
    Repo: google.cloud					      <
    Repo: grafana					      <
    Repo: ibm.qradar					      <
    Repo: intersight-ansible				      |	    Repo: ios
    Repo: junipernetworks.junos				      |	    Repo: iosxr
							      >	    Repo: junos
    Repo: mellanox.onyx					      <
    Repo: netapp					      <
    Repo: netapp					      <
							      >	    Repo: netcommon
    Repo: openvswitch.openvswitch			      |	    Repo: openvswitch
    Repo: ovirt-ansible-collection			      <
    Repo: santricity					      <
    Repo: sensu-go-ansible				      <
    Repo: servicenow-ansible				      <
    Repo: splunk.es					      |	    Repo: vmware
    Repo: vyos.vyos					      |	    Repo: vyos
    Repo: wti-collection				      <

```

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
Thats a chunk of work there, thanks @dmsimard! I've spotted some issues, see the inline comments...

@gundalow any concerns here in general? Any repos we shouldn't index for some reason?

File: config/collections.yml [hunk 1]
```diff
@@ -151,16 +217,24 @@ default:
...
     Org: ansible-collections
-    Repo: community.qradar
-    Regex: ''
-    NewCollection: 'TRUE'
-    MergeKey: ''
+    Repo: community.rabbitmq
   - Site: https://github.com
     Org: ansible-collections
-    Repo: community.rabbitmq
     Regex: rabbitmq
```
Comment (GregSutcliffe) [inline config/collections.yml:223]: This entry has no "Repo" key any more.

Comment (dmsimard) [reply config/collections.yml:221]: Oops, that looks like a copy/paste mistake, fixed in https://github.com/ansible-community/stats-crawler/pull/1/commits/8f02ee497b65534afc5977d060b0d93fcec479f8 thanks !

**Comment by GregSutcliffe** [comment]
Confirmed this parses fine in the crawler:

```
read_collections_yaml() %>% str()
'data.frame':	96 obs. of  6 variables:
```

Just need an ack ;)

**Comment by gundalow** [comment]
Looks good.
Seems like community.asa is an empty repo at the moment.

**Review by gundalow — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Deployed to prod, should be visible from tomorrow.

**Pull request merged**